### ISSUE TITLE
[BUG] Notifications disappear immediately.

### ISSUE THREAD

**Issue by not-night-but** [phuhl/linux_notification_center#248]
Just started trying out deadd, and I'm loving it so far. My only gripe is that notifications get dismissed less than a second after they are received. My default timeout is set to 10000ms and raising that number doesn't seem to have an effect. I've also tried a match/modify rule to force the notification to stay until dismissed, but no joy. This doesn't happen using dunst so I'm assuming it's an issue with deadd or something to do with my setup specifically (if latter is  the case, no worries, I'll soldier on).

I'm on Garuda Linux using bspwm. I installed deadd using the deadd-notification-center-git package in the chaotic-aur.

Thanks in advance for any help you can send my way.

**Comment by phuhl** [comment]
Hi, thanks for the report. When you say dismissed, do you mean the popup disappears?

**Comment by not-night-but** [comment]
@phuhl yeah the popup disappears. Still visible in the notification center, but my expectation was that the popup would stay for the timeout length, unless that assumption was wrong?

**Comment by phuhl** [comment]
That's strange. Is it happening with all notifications? Can you share your config?

**Comment by TheStutterNerd** [comment]
I've just updated to the latest version. Happening to me too.
Same story when using default config.
